Dockerfiles/agent/entrypoint/50-eks.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+if [[ -z "${DD_EKS_FARGATE}" ]]; then
+    exit 0
+fi
+
+# Set a default config for EKS Fargate if found
+# Don't override /etc/datadog-agent/datadog.yaml if it exists
+if [[ ! -e /etc/datadog-agent/datadog.yaml ]]; then
+    ln -s  /etc/datadog-agent/datadog-kubernetes.yaml \
+           /etc/datadog-agent/datadog.yaml
+fi
+
+# Remove all default checks & enable kubelet and eks_fargate checks
+if [[ ! -e /etc/datadog-agent/conf.d/kubelet.d/conf.yaml.default ]]; then
```

Comment:
Why this check in particular? Can it legitimately be customized by the user in eks fargate? If so should the part about the eks_fargate check be in the block or in a separate one?

Reply:
removed